src/tooltip/tooltip.js

```diff
@@ -302,6 +299,18 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
 
             attrs.$observe( prefix+'Title', function ( val ) {
               ttScope.title = val;
+              if (prefix === 'popover') {
+                $timeout(positionTooltip, 0, false);
+              }
+            });
+
+            attrs.$observe( prefix + 'Placement', function () {
+              if (ttScope.isOpen) {
+                $timeout(function () {
+                  prepPlacement();
+                  show()();
```

Comment:
Doesn't this mean that if the placement is changed while the tooltip is hidden, it'll show?